mitm_service/mitm_service.py

```diff
@@ -0,0 +1,83 @@
+import binascii
+import socket
+import struct
+import fcntl
+
+from mitm_service.arp_poison.arp_poison import ARPPoisonService
+from mitm_service.tunneler.tunneler import L2Tunnel
+
+SIOCGIFHWADDR = 0x8927
```

Comment:
???

Reply:
linux ioctl constant, thats actually what its called. Look it up.

Reply:
lol pls add "linux ioctl constant" this as comment